create-featurenet/action.yml

```diff
@@ -123,28 +123,63 @@ runs:
       id: get-ref-properties
       uses: Cardinal-Cryptography/github-actions/get-ref-properties@v1
 
-    - name: Checkout featurenets repo
+    - name: Get node commit SHA
+      if: ${{ inputs.featurenet-aleph-node-image == 'testnet' ||
+        inputs.featurenet-aleph-node-image == 'mainnet' }}
+      id: get-node-commit-sha
+      uses: Cardinal-Cryptography/github-actions/get-node-system-version@v1
+      with:
+        env: ${{ inputs.featurenet-aleph-node-image }}
+
+    - name: Checkout latest featurenets repo
       uses: actions/checkout@v3
       with:
         repository: Cardinal-Cryptography/${{ inputs.repo-featurenets-name }}
         token: ${{ inputs.gh-ci-token }}
         path: "${{ inputs.repo-featurenets-name }}"
         ref: main
 
-    - name: Get node commit SHA
+    - name: Checkout aleph-node repo to get featurenets tag (testnet/mainnet)
       if: ${{ inputs.featurenet-aleph-node-image == 'testnet' ||
         inputs.featurenet-aleph-node-image == 'mainnet' }}
-      id: get-node-commit-sha
-      uses: Cardinal-Cryptography/github-actions/get-node-system-version@v1
+      uses: actions/checkout@v3
       with:
-        env: ${{ inputs.featurenet-aleph-node-image }}
+        repository: Cardinal-Cryptography/aleph-node
+        token: ${{ inputs.gh-ci-token }}
+        path: "aleph-node-testnet-or-mainet"
+        ref: ${{ steps.get-node-commit-sha.outputs.sha }}
+
+    - name: Get compatible featurenet tag
+      id: get-featurenets-tag
+      shell: bash
+      # yamllint disable rule:line-length
+      run: |
+        if [[ "" == 'testnet' || "" == "mainnet" ]]; then
+          aleph_node_dir=${{ github.workspace }}/aleph-node-testnet-or-mainnet
+        else
+          aleph_node_dir=${{ github.workspace }}
+        fi
+        cd "${{ inputs.repo-featurenets-name }}"
+        ./Ops.sh get-compatible-featurenet-tag "${aleph_node_dir}" > tmp-output-fe-tag.txt
+        echo "featurenets-tag=$(cat tmp-output-fe-tag | grep '__output:featurenets-tag:' | cut -d: -f3)" >> $GITHUB_OUTPUT
```

Comment:
Should be `tmp-output-fe-tag.txt` instead `tmp-output-fe-tag`

Reply:
Fixed.